### PR TITLE
Jing local replicate

### PR DESCRIPTION
**Pull request by jeannewu**

### CODE REVIEW

File: .Rhistory
Comment (jacobmpeters) [inline .Rhistory:1]: Please do not include .Rhistory in your commits. Please delete it and add it to your .gitignore file.

File: .gitignore
Comment (jacobmpeters) [inline .gitignore:1]: You also do not need to include the .gitignore file in your commits.

File: prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd
Comment (jacobmpeters) [inline prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd:1]: I don't see any changes that are likely to cause the cloud build/run to fail, so this looks good to me.

**Review by jacobmpeters — CHANGES_REQUESTED** [review]
Please remove the .Rhistory file from your commit. Otherwise this looks fine.

File: prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd
Comment (jeannewu) [reply prod/Consolidated_Weekly_Report_4Pipeline_BQupdated_06082023.Rmd:1]: great. Thanks.

**Review by jacobmpeters — APPROVED** [review]

**Pull request merged**